### PR TITLE
update migration guide to describe solaraviz updates

### PR DESCRIPTION
**Pull request by Corvince**

### CODE REVIEW

**Review by EwoutH — APPROVED** [review]
Awesome, thanks for the write up!

File: docs/migration_guide.md [hunk 1]
```diff
@@ -64,4 +64,67 @@ You can access it by `Model.steps`, and it's internally in the datacollector, ba
...
+#### Plotting text
+
+To plot model-dependent text the experimental SolaraViz provided a `make_text` function that wraps another functions that receives the model and turns its string return value into a solara text component. Again, this other function can now be passed directly to the new SolaraViz components array. It is okay if your function just returns a string.
+
+```python
+# old
+from mesa.experimental import SolaraViz, make_text
+
+def show_steps(model):
+    return f"model.steps"
```
Comment (rht) [inline docs/migration_guide.md:119]: Typo, should have the curly braces.

File: docs/migration_guide.md [hunk 2]
```diff
@@ -64,4 +64,67 @@ You can access it by `Model.steps`, and it's internally in the datacollector, ba
...
+
+SolaraViz(model_cls, model_params, measures=make_text(show_steps))
+
+# new
+from mesa.visualisation import SolaraViz
+
+def show_steps(model):
+    return f"model.steps"
```
Comment (rht) [inline docs/migration_guide.md:127]: Also typo.

**Comment by EwoutH** [comment]
I would like to move this one forward, so that we don't block each other when writing the guide.

**Comment by EwoutH** [comment]
@Corvince do you still want to do the name change to `ModelExplorer`?

If so, this might be the name to do it, before it's everywhere in our docs.

**Comment by Corvince** [comment]
I am not 100% sure on the name still. Lets leave it for now, should be a quick find&replace once we have a better name

**Pull request merged**